documentation/explainers/price-feed-identifiers.md

```diff
@@ -0,0 +1,77 @@
+# Price feed identifiers
+
+This document explains how price feed identifiers are configured and how they are used.
+
+## What is an identifier
+
+An identifier is the term we use to refer to an underlying asset whose price a derivative tracks. Examples of underlying assets
+are the price of Bitcoin in US dollars, the spot price of gold, or the daily high temperature in New York City.
```

Comment:
nit: do you think we should denominate the price of gold as we do with bitcoin, e.g. "the spot price of gold in US dollars"?

Reply:
Done.